### DISCUSSION TITLE
Docker User Permissions

### DISCUSSION THREAD

**Discussion by tessamerrill** [open-webui/open-webui#6689] [Ideas]
# Feature Request

**Is your feature request related to a problem? Please describe.**
I'm running Open-WebUI on Unraid, which when using Docker, the best practice is to use User 99 and Group 100 (nobody) for filesystem access. Open-WebUI sets these values to root:

https://github.com/open-webui/open-webui/blob/7228b39064ac28e1240bf8998f2a35535c6f7ef5/Dockerfile#L19-L21

**Describe the solution you'd like**
1) Allow the user to pass UID/GID into the container
2) Give instructions to build the image (and changing the Dockerfile) to change the user permissions
3) Build and push an Unraid-specific image (least viable option, imho)

**Describe alternatives you've considered**
I've been trying to build the image on my own, however I cannot find the right way to build it (as `docker build -t open-webui:ollama .` doesn't seem to work right.)

**Additional context**
This could be more viable as a synced fork, which I could look into.

**Comment by voc0der** [comment]
Did this ever go anywhere? Seems not great to store all of the files as root.
